opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -53,7 +40,7 @@ public FilterRegistrationBean tracingFilter() {
     }
 
     @Bean
-    public WebMvcConfigurerAdapter tracingHandlerInterceptor() {
+    public WebMvcConfigurerAdapter tracingHandlerInterceptor(final Tracer tracer) {
```

Comment:
I'd remove this "final" to keep things consistent - arguments in other methods are not final